sql/parser/parse_test.go

```diff
@@ -84,6 +84,8 @@ func TestParse(t *testing.T) {
 		{`DROP TABLE a.b`},
 		{`DROP TABLE a, b`},
 		{`DROP TABLE IF EXISTS a`},
+		{`DROP INDEX a.b@c`},
```

Comment:
Yeah, this is actually a bigger problem than that. In postgres, DROP INDEX can refer to index names globally (and index names are globally unique). We're probably going to need to address this on a larger scale, along with some other constraint vs index reform.